### ISSUE TITLE
setWeaponCapacity doesn't work correctly

### ISSUE THREAD

**Issue by Fernando9995** [altmp/altv-issues#1922]
### Description of the problem

I set vehicle weapon ammo 3
I shoot all ammo
Current ammo is 0
I set again ammo 3
The ammo is 3 on server, but i can't shoot anymore
I noticed that such a problem when setting the same values. If set ammo 3, shoot all of them and then set a number other than last (3 in my case), all will be fine.
The same problem is solved if you fly out and fly back into the vehicle stream

![image](https://github.com/altmp/altv-issues/assets/129927256/1ec37065-b058-4617-8e02-fb62915488a4)

### Reproduction steps

client-side
```
alt.on("keydown", (key) => alt.emitServer("keydown", key));
```
server-side
```
alt.onClient("keydown", async (player, key) => {
  if (key === 72) { // 72 = H
    const vehicle = new alt.Vehicle("rhino", player.pos.x, player.pos.y, player.pos.z, 0, 0, player.rot.z);
    player.setIntoVehicle(vehicle, 1);
  } else if (key === 73) { // 73 = I 
    const vehicle = player.vehicle;
    if (vehicle == null) {
      return;
    }

    vehicle.setWeaponCapacity(0, 5);
  }
})
```

### Expected behaviour

Vehicle weapon ammo must be installed correctly

### Operating system

Windows 10 22H2

### Version

15.0-dev397 (dev)

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by FabianTerhorst** [comment]
Yes thats known and a issue in release as well. You can only set a value that is different otherwise gta ignores it.
